skbio/tree/_tree.py

```diff
@@ -2705,6 +2705,47 @@ def compare_tip_distances(self, other, sample=None, dist_f=distance_from_r,
         return dist_f(self_matrix, other_matrix)
 
     @experimental(as_of="0.4.0")
+    def bifurcate(self, insert_length=None, remove_singles=True):
```

Comment:
Unsure, but here's the general [case](https://github.com/pycogent/pycogent/blob/master/cogent/core/tree.py#L1417). I confess, I thought we'd ported this code but it doesn't appear to be in skbio...